### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vinaya563**
## Related Issue
- Issue Number (#1195 )

## Proposed Changes
-Added Readme  and description about  Circular Linked List and its applications in various domains .

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [ ] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Lakhankumawat** [comment]
Make Changes inside `[L-LinkedList/C-CircularLinkedList/README.md]` and add code for it too ...

You need to modify your readme further 
1. Make sure headlines are short and follow camel casing
2. Add a working image of program ( not code or ouput screenshot)
3. Restructure README navigation links and organise it 
4. Follow the guidelines given below

You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Comment by vinaya563** [comment]
I have made the changes as said !

**Comment by Lakhankumawat** [comment]
> I have made the changes as said !

Nope Not made all the changes yet

**Comment by vinaya563** [comment]
Now is it okay ?
If not can you please say what exactly should I change ?

File: L-LinkedList/C-CircularLinkedListkedList/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+
+### [Circular Linked Lists](https://github.com/Lakhankumawat/LearnCPP/tree/main/L-LinkedList/C-CircularLinkedList)
+
+
+
+
```
Comment (Lakhankumawat) [inline L-LinkedList/C-CircularLinkedListkedList/README.md:6]: make the link to headlines you covered in README over here

See here
(https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md)

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by Lakhankumawat** [comment]
No Updates ?

**Comment by vinaya563** [comment]
Now is it fine ?
or any other changes are required ?

**Review by Lakhankumawat — APPROVED** [review]
LGTM 🎉✨🐱‍👤

**Review by Lakhankumawat — COMMENTED** [review]
Since there were many minor changes required .

**Comment by Lakhankumawat** [comment]
@all-contributors please add @vinaya563 code ,

**Pull request merged**